json_serializable/test/src/_json_serializable_test_input.dart

```diff
@@ -120,6 +120,21 @@ class FinalFieldsNotSetInCtor {
   FinalFieldsNotSetInCtor();
 }
 
+@ShouldGenerate(r'''
+SetSupport _$SetSupportFromJson(Map<String, dynamic> json) {
+  return SetSupport((json['values'] as List)?.map((e) => e as int)?.toSet());
+}
+
+Map<String, dynamic> _$SetSupportToJson(SetSupport instance) =>
+    <String, dynamic>{'values': instance.values?.toList()};
```

Comment:
I have not changed return type cast (from `toList` to `toSet`) as `Set` is not supported by default by `dart:convert`

Reply:
yup